app/serializers/merchant_serializer.rb

```diff
@@ -0,0 +1,4 @@
+class MerchantSerializer
+  include FastJsonapi::ObjectSerializer
+  attributes :name
+end
```

Comment:
The serializer has to have this information. We don't need :id or describe has_many relationships. Doesn't have to follow our model.
```
{
  "data": {
    "id": "1",
    "type": "merchant",
    "attributes": {
      "name": "Store Name"
    }
  }
}
```